cfme/tests/automate/test_vmware_methods.py

```diff
@@ -23,7 +23,7 @@
 
 def pytest_generate_tests(metafunc):
     argnames, argvalues, idlist = testgen.provider_by_type(
-        metafunc, ['virtualcenter'], 'provisioning')
+        metafunc, ['virtualcenter'], required_fields=['provisioning'])
```

Comment:
Further down though I see `'provisioning', 'ci-template'` which means we possibly have a `'provisioning'` collection with only `'ci-template'` and no `'template'`. Is this correct?